src/TextField/TextField.jsx

```diff
@@ -467,7 +468,6 @@ const TextField = React.createClass({
         htmlFor={inputId}
         shrink={this.state.hasValue || this.state.isFocused}
         disabled={disabled}
-        onTouchTap={this.focus}
```

Comment:
It was introduced by https://github.com/callemall/material-ui/pull/2228.
Is the browser support of `pointerEvents` enough to remove this line?